SQLtoys/SQLtoys

```diff
@@ -0,0 +1,9 @@
+create table toys (id int, name varchar(100), price float, stock int);
+insert into toys values (1, 'Car', 5.00, 40);
+select * from toys;
+alter table toys add column department_id;
+insert into toys values (2, 'GI Joe', 10.00, 40, 101);
```

Comment:
you set the department id on the toys records before that department existed in the database. not a big deal here, but if you had had referencial integrity checks, this would have failed.